code/api/src/seeders/formal.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('surveys', [
+      {
+        type: 'bottom',
+        image: 'public/images/survey/formal/mens_pants_formal.jpg',
+        score: [5, 0],
```

Comment:
I think this should be the opposite? [0,5]

Reply:
Man, apparently I can't read a graph anymore...fixed.